bin/modulizer.js

```diff
@@ -18,12 +18,34 @@
 
 process.title = 'modulizer';
 
+/*
+ * NOTE: The contents of this file should work on as many version of Node.js
+ * as possible. This means it *can not* use any >ES5 syntax and features.
+ * Other files, which may use >=ES2015 syntax, should only be loaded
+ * asynchronously after this version check has been performed.
+ */
+
+var semver = require('semver');
+var version = require('../package.json').engines.node;
+
+// Exit early if the user's node version is too low.
+if (!semver.satisfies(process.version, version)) {
+  // Strip version range characters leaving the raw semantic version for output
+  var rawVersion = version.replace(/[^\d\.]*/, '');
```

Comment:
The engines field might change to be something with a more complex format. It's fine to just report that string, something like:

`'Polymer Modulizer requires Node version ' + engines.node + '. You have...`

Reply:
to clarify: remove `rawVersion` formatting and just log the raw `engines.node` string

Reply:
Sorry, I misread your comment. Updating. 👍